basic/exercise11/scriptSpec.js

```diff
@@ -34,4 +34,12 @@ describe("Conversion does not lose precision when it is converted and converted
         var degreesFahrenheit = convertTemperatureFromCelsiusToFahrenheit(degreesCelsius);
         expect(degreesFahrenheit).toBe(0);
     });
+});
+
+describe("Conversion does not lose precision when it is converted and converted back:", function () {  
```

Comment:
Why do you create new describe section? It is the same as previously creacted, so you may add it function call directly to this previous section.